### PR TITLE
Round out remaining docker images

### PR DESCRIPTION
**Pull request by msheiny**
This should round out the rest of the docker images we currently are hosting on `quay.io`. Let me know what else you'd like me to try and mix in at this time ;)

### CODE REVIEW

File: logstash-verifier/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
+FROM msheiny/debian-jessie-systemd:latest
```
Comment (conorsch) [inline logstash-verifier/Dockerfile:1]: Can I get the `msheiny/debian-jessie-systemd` image in here too? I'll handle converting and standardizing on the FPF Quay registry, while adding the timestamp tags.

Comment (conorsch) [reply logstash-verifier/Dockerfile:1]: Will handle msheiny -> freedomofpress for systemd container as a separate issue.

**Review by conorsch — APPROVED** [review]
Solid progress. Subsequently we'll add timestamp tagging and a build wrapper script.

File: logstash-verifier/Dockerfile (hunk 1 above)
Comment (msheiny) [reply logstash-verifier/Dockerfile:1]: oh whoops -- i didnt realize that was tagged in the dockerfile here, should be an easy switch-over to our FPF quay hosted one

**Pull request merged**